collection/holly.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "holly",
+    "meaning": "to prick, or refers to the holly shrub",
+    "aliases": [
+        "hollie",
+        "holley",
+        "holli"
+    ],
+    "translations": {
```

Comment:
Hi @Holly-E,
Thanks for your PR. Can you please make sure that these translations are the actual name translations in other languages and not the translations for the meaning of the name?

For example name holly might translate to `holley`, `hollie` or it may not translate at all and stay as `holly` in different languages. But you wouldn't call someone named holly in German llex would you?